code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -11,17 +11,14 @@
 	name = "bed"
 	desc = "This is used to lie in, sleep in or strap on."
 	icon_state = "bed"
-	var/mob/living/buckled_mob
+	can_buckle = 1
+	buckle_lying = 1
 
 /obj/structure/stool/bed/alien
 	name = "resting contraption"
 	desc = "This looks similar to contraptions from earth. Could aliens be stealing our technology?"
 	icon_state = "abed"

```

Comment:
It was used, but it was removed at some point. Also the sprite is shit and it has no reason to exist.